lib/chai/utils/proxify.js

```diff
@@ -64,7 +64,7 @@ module.exports = function proxify (obj, nonChainableMethodName) {
         }
       }
 
-      return target[property];
+      return Reflect.get(target, property);
```

Comment:
I have an idea how to improve proxies performance by much, utilizing third parameter to `Reflect.get`, so I think we should leave it like this.

PS: much pleasure to review such careful commits 👍